modules/SofaMiscFem/src/SofaMiscFem/TriangularFEMForceField.inl

```diff
@@ -1512,10 +1463,15 @@ void TriangularFEMForceField<DataTypes>::draw(const core::visual::VisualParams*
         vparams->drawTool()->drawTriangles(vertices,colorVector);
         vertices.clear();
         colorVector.clear();
+        vertexInfo.endEdit();
     }
 
     if (showFracturableTriangles.getValue())
```

Comment:
yes, even if I couldn't see one during the tests. I only remove the default FEM rendering which was showing green/blue triangles without meaning. 
Now by default if you render this FEM it will show the stressMap value. But you can still uncheck this rendering and thus it will render nothing.